src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -106,6 +106,10 @@ s! {
         pub f_uid_uuid: ::uuid_t,
     }
 
+    #[deprecated(
+        since = "0.2.106",
+        note = "stat.st_blksize is an i64 and replaces stat.st_qspare1 in DragonFly 5.8"
```

Comment:
I think something like this is clearer:

```suggestion
        note = "stat.st_blksize is an i64 and stat.st_qspare1 is replaced with stat.st_blksize in DragonFly 5.8"
```